packages/x-tailwindcss/demo/src/components/xds-facet-filter.vue

```diff
@@ -97,10 +100,13 @@
         ),
         Ghost: this.colors.map(addParentClasses(this.base, this.ghost)),
         'Selected Ghost': this.colors.map(addParentClasses(this.base, this.ghost, this.selected)),
+        Simple: this.colors.map(addParentClasses(this.base, this.simple)),
+        'Selected Simple': this.colors.map(addParentClasses(this.base, this.simple, this.selected)),
         Disabled: [
           this.base,
           addParentClasses(this.base)(this.selected),
-          addParentClasses(this.base)(this.ghost)
+          addParentClasses(this.base)(this.ghost),
+          addParentClasses(this.base)(this.simple)
```

Comment:
What do you think of adding the `underline` and the `selected` variants too?

```suggestion
          addParentClasses(this.base, this.selected)(this.ghost),
          addParentClasses(this.base)(this.simple),
          addParentClasses(this.base, this.selected)(this.simple),
          addParentClasses(this.base)(this.underline),
          addParentClasses(this.base, this.selected)(this.underline)
```

Testing this change, I noticed the `disabled` styles are not properly applied on `hover` for the `underline` and the `simple+selected` variants:

![image](https://user-images.githubusercontent.com/77450928/231399394-c84b24c0-4ff3-465a-b464-775fa4c7ed15.png)